src/main/sync-root-folder/handlers.ts

```diff
@@ -9,8 +9,10 @@ ipcMain.handle('get-sync-root', () => {
 
 ipcMain.handle('choose-sync-root-with-dialog', chooseSyncRootWithDialog);
 
-ipcMain.handle('open-sync-folder', () => {
+ipcMain.handle('open-sync-folder', async () => {
   const syncFolderPath = configStore.get('syncRoot');
 
-  return shell.openPath(syncFolderPath);
+  const errorMessage = await shell.openPath(syncFolderPath);
+
+  if (errorMessage) throw new Error(errorMessage);
```

Comment:
We cannot throw errors on the handlers https://github.com/electron/electron/issues/24427